config.yaml

```diff
@@ -58,3 +58,10 @@ options:
     description: >
       When set to 'insecure', the charm test mode is activated, which may deactivate some security
       hardening measures.
+  allowlist:
+    type: string
+    default: "0.0.0.0/0:1-65535, 0.0.0.0/0:1-65535:udp"
+    description: >
+      A list of IPv4 networks and tcp/udp ports that runners can access. Note that ICMP traffic 
+      and IPv6 traffic are always disallowed. Additionally, the runner will always have
+      access to essential services such as DHCP and DNS regardless of the allowlist configuration.
```

Comment:
What is the logic behind allowing everything in IPv4 by default from a security perspective?

Reply:
For the default value, we only have two options: allow all or allow none. But when the charm is set to allow none, the GitHub runner can't even start, so I opt to chose allow all as the default value.

Reply:
Makes sense, thanks!